ConfCore/TranscriptIndexer.swift

```diff
@@ -102,7 +102,7 @@ public final class TranscriptIndexer {
         )
 
         do {
-            let realm = try storage.makeRealm()
+            let realm = try storage.makeRealm(on: nil)
```

Comment:
This might actually supposed to be `storage.realm` instead. At this point in the code it is on the XPC thread which is where the underlying realm was created.